### PR TITLE
feat: Initial configuration and Story map tests

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Added initial configuration for E2E tests
- Added Story map tests
- Added Github action configuration
- Added Docker local configuration

### Checklist
- [x] Corresponding issue has been opened

### Related Issues
Fixes #https://github.com/techmatters/terraso-web-client/issues/819

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* global process */
+import { v4 as uuidv4 } from 'uuid';
```
Comment (paulschreiber) [inline src/config.js:2]: question is there a reason to alias this?

**Review by paulschreiber — APPROVED** [review]

File: src/config.js (hunk 1 above)
Comment (josebui) [reply src/config.js:2]: @paulschreiber For me this is clearer to understand in its usage, if we just leave it as v4 then the usage will be `v4()` and that is really confusing from a redability point of view

**Review by garobrik — COMMENTED** [review]
this is so cool!!

question: how come this is implemented in a separate repo?

**Comment by josebui** [comment]
> do we need to create specific test users or do anything specific to handle them or could we just create normal users?

@shrouxm From my experience and understanding of e2e tests, they should not be hooked directly to the code changes, some of my reasons for this are:

- E2E tests are expensive, in terms of execution and cost(free right now since we are under the github free restrcitions), so in terms of execution time if you check the current executions just running 3 tests is taking around 3 minutes, so it is not a process that we want to verify for every commit added, that is why we have unit tests.
- E2E tests in large projects are not managed by the development team, they are managed and executed by the QA team so they need some level of isolation to evolve separately.

**Pull request merged**